packages/gatsby-source-contentful/src/generate-options-table.js

```diff
@@ -0,0 +1,13 @@
+import table from "markdown-table"
```

Comment:
Generating the markdown for the README is nice and maybe we do that as a secondary piece of support. But it'd be great if there was a CLI command equivalent to `--help` for gatsby plugin add. That way you don't need to look up docs at all, and the information is available on demand.

Reply:
Good idea, we should definitely do that in the future!